user/makefile

```diff
@@ -3,11 +3,7 @@ USER_MODULE_PATH=.
 
 BUILD_PATH_EXT = $(USER_BUILD_PATH_EXT)
 
-DEPENDENCIES = wiring system services communication hal
-
-ifdef TEST
-DEPENDENCIES += crypto
-endif
+DEPENDENCIES = wiring system services communication hal crypto
```

Comment:
I had to add this dependency because the system leaks tons of internal definitions to the app's global namespace. I'll look how I can work around that differently :+1:

Reply:
Fixed.